package.json

```diff
@@ -35,13 +35,13 @@
     "axios": "~0.21.0",
     "expo": "~42.0.0",
     "expo-application": "~3.2.0",
+    "expo-cli": "^6.0.6",
```

Comment:
This was installed globally, I think it's better to manage this dependency in package.json.